### PR TITLE
tests: add check that apparmor.service service works correctly with vendored apparmor

### PR DESCRIPTION
**Pull request by mvo5**

### CODE REVIEW

**Review by MiguelPires — APPROVED** [review]

File: tests/main/snapd-snap/task.yaml [hunk 1]
```diff
@@ -155,6 +155,9 @@ execute: |
       if ! os.query is-trusty && ! os.query is-xenial; then
         systemctl status snapd.apparmor.service
       fi
+      if os.query is-ubuntu; then
+        systemctl status apparmor.service
```
Comment (sergiocazzolato) [inline tests/main/snapd-snap/task.yaml:159]: When I was researching the issue, initially the apparmor.service was active, but when I restarted it it failed, could be  better to restart it and check it is active after that?

**Comment by alexmurray** [comment]
In general the snapd-snap:lxd spread test is failing due to kill timeout - likely the VMs are running out of memory - so I think we may need to bump the memory for this test.

File: tests/main/snapd-snap/task.yaml (hunk 1 above)
Comment (sergiocazzolato) [reply tests/main/snapd-snap/task.yaml:159]: Also, this test is not being executed in ubuntu-core which is where we reproduced the issue. Perhaps we could have a new test so we also validate the beta images.

**Review by olivercalder — APPROVED** [review]
Thanks! This will be a great sanity check to have.

**Pull request merged**